examples/last-10-builds-async.php

```diff
@@ -7,14 +7,14 @@
 use WyriHaximus\React\GuzzlePsr7\HttpClientAdapter;
 use WyriHaximus\Travis\BuildCollection;
 use WyriHaximus\Travis\Builds;
-use WyriHaximus\Travis\Client;
```

Comment:
@WyriHaximus 

But this one actually intended to rename `Travis\Client` to `Travis\HttpClient`